connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtocolSuite.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.connect.planner
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.connect.proto
+import org.apache.spark.sql.catalyst.expressions.AttributeReference
+import org.apache.spark.sql.catalyst.plans.PlanTest
+import org.apache.spark.sql.catalyst.plans.logical.{LocalRelation, LogicalPlan}
+import org.apache.spark.sql.sparkconnect.plans._
+import org.apache.spark.sql.test.SharedSparkSession
+import org.apache.spark.sql.types.IntegerType
+
+class SparkConnectProtocolSuite extends SparkFunSuite
+  with SharedSparkSession with SparkConnectPlanTest with PlanTest {
+
+  val testRelation = LocalRelation(AttributeReference("a", IntegerType, nullable = true)())
```

Comment:
LocalRelation is a weird thing it's very unlike what we typically deal with in Spark Connect as it has a full schema instead of an unresolved schema. Since we don't have support for it yet, what about teaching the fixture to create a temp table that we use instead to avoid adding two concepts in one patch?

Reply:
I see. That makes sense. We can stay on registering temp table thus do not need to add LocalRelation for now.


Eventually though we have to have this `LocalRelation` with an extra value fields because many usage of `Seq(x, y, z).toDF` will create a LocalRelation with `x, y, x` values as payload.   This usage is common for development purpose.

Reply:
For me given that this pattern (`Seq(x, y, z).toDF`) is common, support `LocalRelation` is useful in the future, it's ok for me to support it in protobuf.

However if you prefer to simplify our protobuf for now before the init PR, we can do it and add `LocalRelation` in the future.